package.json

```diff
@@ -22,7 +22,7 @@
     "test-webworker": "browserify --no-commondir --full-paths -p [ mocaccino -R spec --color ] test/webworker/webworker-support-assessment.js | phantomic --web-security false",
     "test": "run-s test-node test-headless test-webworker",
     "check-dependencies": "dependency-check package.json --unused --no-dev",
-    "build": "./build.js",
+    "build": "node ./build.js",
```

Comment:
This change was necessary to run `npm build` or `npm test` locally on a Windows box.  If it's inappropriate, then I'll happily revert this change.

Reply:
That's fine. At some point we agreed that it was not a pre-requisite that the entire build process runs on Windows, as none of the maintainers were using it, but we are certainly not opposed to improving the situation :-)